### PR TITLE
Remove Spaces from API key. 

### LINKED ISSUE #78: Issue with Termux not reading API keys properly
I've been running into a recent issue where Termux cannot properly read the API key.

It looks like Termux is adding empty space to the API key. I'm going to have to experiment with fixing this issue.

### PR DESCRIPTION
**Pull request by BenjaminUrquhart**
This should (in theory) remove any extra white space from the API key. In the setup file; the key is shown to the user in single quotes in order to spot trailing white space (can be removed if needed).
Fixed #78

### CODE REVIEW

**Comment by o355** [comment]
PR looks good.

**Comment by o355** [comment]
PR looks good, and I'll try and use the review feature to see if I can comment on a few code oddities that I saw.

File: setup.py [hunk 1]
```diff
@@ -759,21 +759,21 @@ def printException_loggerwarn():
 "Press any key when you are done and ready.", sep="\n")
 input()
 print("Please input your API key below.")
-apikey_input = input("Input here: ")
+apikey_input = str(input("Input here: ")).replace(" ","").replace("\n","").replace("\t","")
 logger.debug("apikey_input: %s" % apikey_input)
-print("", "Just to confirm, the API key you gave me was: " + apikey_input
-      + ".", sep="\n")
+print("", "Just to confirm, the API key you gave me was: '" + apikey_input
+      + "'", sep="\n")
```
Comment (o355) [inline setup.py]: Is there any reason to add `'` on top of `"`.

The single quotation mark shouldn't replace the period in some places as it's part of the setup dialogue.

File: setup.py [hunk 2]
```diff
@@ -759,21 +759,21 @@ def printException_loggerwarn():
...
     while True:
         logger.debug("User now re-entering key...")
         print("","Please input your API key below.", sep="\n")
-        apikey_input = input("Input here: ")
+        apikey_input = str(input("Input here: ")).replace(" ","").replace("\n","").replace("\t","")
         logger.debug("apikey_input: %s" % apikey_input)
-        print("Just to confirm, the API key you gave me was: " + apikey_input
-              + ".")
+        print("Just to confirm, the API key you gave me was: '" + apikey_input
+              + "'")
```
Comment (o355) [inline setup.py]: Same goes here, the period is part of the dialogue. I think we only need a replace statement and not necessarily change to the setup dialogue.

File: setup.py [hunk 3]
```diff
@@ -14,6 +14,7 @@
 |               |           @ # .`
 '''
 
+
```
Comment (o355) [inline setup.py]: Remove empty line

Comment (BenjaminUrquhart) [reply setup.py]: I referenced the wrong issue in the initial fix, so I had to make a change to fix it.

**Comment by BenjaminUrquhart** [comment]
Alright, I removed the single quotes and the extra line.

**Comment by o355** [comment]
Thanks for making the changes requested. I'll review the code again and merge it if all looks good.

**Pull request merged**